dlp/snippets/src/test/java/dlp/snippets/InspectTests.java

```diff
@@ -416,4 +475,29 @@ public void testInspectStringAugmentInfoType() throws Exception {
     assertThat(output).contains("Findings: 1");
     assertThat(output).contains("Info type: PERSON_NAME");
   }
+
+  @Test
+  public void testInspectWithStoredInfotype() throws Exception {
+    // Create a new stored InfoType.
+    String infoTypeId = UUID.randomUUID().toString();
+    createStoredInfoType(PROJECT_ID, GCS_PATH, infoTypeId);
+
+    // Wait for the creation of Stored InfoType.
+    Thread.sleep(30000);
+
+    String textToDeidentify =
+        "My email address is gary@example.com.";
+    InspectWithStoredInfotype.inspectWithStoredInfotype(PROJECT_ID, infoTypeId, textToDeidentify);
+    String output = bout.toString();
+    assertThat(output).contains("Findings: 2");
```

Comment:
what are the two findings in that text? Can you assert the matching infotypes and quotes as well?